### PR TITLE
core: version-chooser: expose dockerhub errors to user

### LINKED ISSUE #757: core: version-chooser: Export dockerhub errors for user
We should inform user when something goes wrong while fetching images.
As:

```
/home/pi/services/versionchooser/utils/dockerhub.py:80: UserWarning: Error status 429
  warn(f"Error status {resp.status}")
CRITICAL:root:error fetching online tags: Failed getting sha from DockerHub!
```

429 Too Many Requests

### LINKED ISSUE #765: core: frontend: version-chooser: Add error message when dockerhub is offline

### PR DESCRIPTION
**Pull request by Williangalvani**
Screenshot from 2022-02-03 11-57-07

Fix #765 
Fix #757

image building to williangalvani/companion-core:version-chooser-errors

### CODE REVIEW

**Review by patrickelectric — APPROVED** [review]

**Comment by Williangalvani** [comment]
@rafaellehmkuhl can you test #390 with this?

**Comment by rafaellehmkuhl** [comment]
> @rafaellehmkuhl can you test #390 with this?

Don't know if its the best way to test it, but refreshing the page and disconnecting the computer from internet left version-chooser on an eternal reload state:

image

When I reconnect the internet, both local and remote are fetched.

File: core/services/versionchooser/frontend/index.html [hunk 1]
```diff
@@ -233,8 +237,10 @@ <h3>Applying...</h3>
         loadAvailableversions() {
           // TODO: validate repository, treat error fetching
           this.loadingImages = true
+          this.error = null
           axios.get(`v1.0/version/available/${this.selectedImage}`).then((response) => {
```
Comment (rafaellehmkuhl) [inline core/services/versionchooser/frontend/index.html:241]: It seems this request would benefit from a timeout.

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

**Comment by Williangalvani** [comment]
>  the computer from internet 

I see. so you actually lost connection to companion, and not companion to the internet.

File: core/services/versionchooser/frontend/index.html (hunk 1 above)
Comment (Williangalvani) [reply core/services/versionchooser/frontend/index.html:241]: I'll include it in the the vuetify rewrite.

**Comment by rafaellehmkuhl** [comment]
> > the computer from internet
> 
> I see. so you actually lost connection to companion, and not companion to the internet.

Yup. Just tested with just internet lost and it works as expected!

image

**Review by rafaellehmkuhl — APPROVED** [review]

**Pull request merged**